easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422' 
+
+homepage = 'https://colibread.inria.fr/software/commet/' 
+description = """ COMMET (“COmpare Multiple METagenomes”) provides a global similarity overview between all datasets of a large metagenomic project. 
+""" 
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# there is no tarball provided, only git checkout through: 
+# https://github.com/pierrepeterlongo/commet/archive/master.zip
+sources = [SOURCE_ZIP] 
+files_to_copy = ["bin","doc","include","ABCDE_bench",(["Commet.py"],'bin')] 
+
+pythonversion = '2.7.11' 
+versionsuffix = '-Python-%(pythonversion)s' 
+
+dependencies = [
+    ('Python', pythonversion), 
+] 
+
+checksums = ['5fe63369ec3722e9e758b7bc1dcdfef3'] 
```

Comment:
move this up right below source please, and definitely mention to which commit this relates to